### DISCUSSION TITLE
Background color for Consumer Group

### DISCUSSION THREAD

**Discussion by n-i-c-o-l-a-s** [tchiotludo/akhq#941] [Q&A] [answered]
Hello. What is the rule to choose the color of the background consumer group please ? 

Sometimes it's green or it's orange.

It doesn't seems to depend on the lag since we have orange color with zero lag.

![image](https://user-images.githubusercontent.com/44503667/145066324-7d3ce9ea-8479-4bc6-817c-bcfdee689da9.png)

Thanks a lot for clarify.

**Comment by tchiotludo** [comment, marked as answer]
The response is here 
https://github.com/tchiotludo/akhq/blob/17c541fb25f0b643f1588a02d308d36450cd042e/client/src/containers/ConsumerGroup/ConsumerGroupList/ConsumerGroupList.jsx#L96-L106

In resume : 
- **GREEN**: Stable 
- **BLUE**: Rebalancing consumer group
- **YELLOW**: All the others case, mostly empty consumer group
